Styles/main.css

```diff
@@ -57,18 +57,21 @@ footer {
     margin-bottom: 2em;
 }
 header {
-    float: right;
-    border: 4px solid rgba(0,0,0,0.1);
+    border-top: 4px solid rgba(0,0,0,0.1);
     margin-left: -2em;
     margin-right: -2em;
 }
+header nav {
+    overflow: auto;

```

Comment:
This declaration causes `<nav>` to include the height of its floated children when determining its own height